### PR TITLE
Milestone 4 complete

### PR DESCRIPTION
**Pull request by OlukaDenis**
Added the operate and calculate helper methods

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import BigJs from 'big.js/big';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let result = null;
+
+  switch (operation) {
+    case '+':
+      result = BigJs(numberOne).plus(BigJs(numberTwo)).toString();
```
Comment (shshamim63) [inline src/logic/operate.js:8]:
```js
BigJs(numberOne).op(BigJs(numberTwo)).toString();
```
Is actually repetitive. So I think you can declare two variables that will contain numberOne and numberTwo.
```js
const firstNumber = BigJs(numberOne);
const secondNumber = BigJs(numberTwo);
```
then you can use them without repetition.

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  if (buttonName === 'AC') {
+    total = null;
+    next = null;
+    operation = null;
+  }
+
+  if (buttonName === '+/-') {
+    total = (total * (-1)).toString();
+    next = (next * (-1)).toString();
+  }
```
Comment (shshamim63) [inline src/logic/calculate.js]: Both total and next should not change. You should decide on which one you need to alter the sign.

**Review by shshamim63 — CHANGES_REQUESTED** [review]
## Status :recycle: Required Changes :recycle: 
___
Hi @OlukaDenis :wave: 
You are almost done with the milestone. Just one major thing is blocking me to approve this right now. Let's resolve it.
___
### :spades: Mandatory Changes :spades: 
___
- You have to decide on which point you are pressing the '+/-' button. Based on that you are going to alter the sign. For example, 
`6 + 5 +/- = should output 1` cause here 5 is changing the sign. But you are changing both.  
___
### :clubs: Optional Changes :clubs: 
___
- I have found some repetition in operate.js. Dry code is always better. Check out the comment.
___
#### Please fix the above-mentioned issue and another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Review by Oghenebrume50 — APPROVED** [review]
### APPROVED

Hi @OlukaDenis kudos for making all changes as requested please merge and move on to the next things, **this is approved** :slightly_smiling_face: 


**RAPHAEL NORIODE**

**Pull request merged**